src/css/base.scss

```diff
@@ -1,18 +1,125 @@
 * {
   margin: 0;
+  font-family: 'Noto Sans', sans-serif;
 }
 
-img {
-  height: 100px;
-  width: auto; 
+// @import 'styles';
+@import '~js-datepicker/src/datepicker';
+$darkGray: darkgray;
+
+@mixin flex-row() {
+  display: flex;
+  flex-direction: row;
```

Comment:
Right on with the mixins! If you want you could shorten this one. Since the default flex direction is row, it'll have `flex-direction: row` even if you don't explicitly put that in there.